dev-libs/libmspack/libmspack-0.6~alpha.recipe

```diff
@@ -88,4 +123,6 @@ INSTALL()
 TEST()
 {
 	make check
+	cd test
+	./cabd_test
```

Comment:
Is `cabd_test` not automatically run by `make check`?

Reply:
Nope.
(I just checked this again to make sure.)

BTW, Are OK with sticking to the stable release and keeping Gentoo's patch?